docs/writing-stories/naming-components-and-hierarchy.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Naming components and hierarchy'
+---
+
+The title of the component you export in the `default` export controls the name shown in the sidebar.
+
+```js
+
+// Button.stories.js
+export default {
+  title: ‘Button’
+}
+```
+
+Yields this:
+
+<div style="background-color:#F8FAFC">
+TODO: per screenshot spreadsheet add image Default ordering of Button with the placement
+<div>
+
+
+### Grouping
+
+It is also possible to group related components in an expandable interface in order to help with Storybook organization. To do so, use the `’/’` as a separator:
+
+```js
+// Button.stories.js
+
+export default {
+  title: ‘Design System/Atoms/Button’
+}
+```
+
+
+```js
+// Checkbox.stories.js
+
+export default {
+  title: ‘Design System/Atoms/Checkbox’
+}
+```
+
+Yields this:
+
+<div style="background-color:#F8FAFC">
+
+TODO: per screenshot spreadsheet add image Show Button ordering/grouping with Design System/Atoms/Button title
+
+<div>
+
+### Roots
+
+By default the top-level grouping will be displayed as a “root” in the UI (the all-caps, non expandable grouping in the screenshot above). If you prefer, you can [configure Storybook](../configure/user-interface#roots) to not show roots.
+
+We recommend naming components according to the file hierarchy. 
+
+### Sorting stories
+
+By default, stories are sorted in the order in which they were imported. This can be overridden by adding `storySort` to the `options` parameters in your `preview.js` file.
+
+The most powerful method of sorting is to provide a function to `storySort`. Any custom sorting can be achieved with this method.
+
+```js
+export const parameters = {
+  options: {
+    storySort: (a, b) =>
+      a[1].kind === b[1].kind ? 0 : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }),
+  },
+};
+```
+
+The `storySort` can also accept a configuration object.
+
+```js
+
+export parameters = {
+  options: {
+    storySort: {
+      method: 'alphabetical', // Optional, defaults to 'configure'.
+      order: ['Intro', 'Components'], // Optional, defaults to [].
+      locales: 'en-US', // Optional, defaults to system locale.
+    },
+  },
+};
+```
+<div>
+TODO: ask tom/dom if we could add table here that could be further expanded
```

Comment:
Seems like a good idea.

Reply:
i've just committed the table with a simple structure that we can probably use. Take a look at it and let me know

Reply:
Looks good!